### PR TITLE
UI/Qt: Switch to the new logo

### PR DESCRIPTION
**Pull request by circl-lastname**
Not done for macOS yet

### CODE REVIEW

File: Ladybird/Qt/ladybird.qrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE RCC><RCC version="1.0">
     <qresource>
-        <file>../Icons/ladybird.png</file>
```
Comment (ADKaster) [inline Ladybird/Qt/ladybird.qrc:3]: This feels pretty weird. Would it be better to copy the icon over the old name instead?

Comment (circl-lastname) [reply Ladybird/Qt/ladybird.qrc:3]: I could symlink the two perhaps

Comment (circl-lastname) [reply Ladybird/Qt/ladybird.qrc]: Is this OK?

Comment (ADKaster) [reply Ladybird/Qt/ladybird.qrc]: Hmmm I suppose. But in theory, we should provide more than one icon size. 

We're not quite using QIcon directly, I think, by providing it only one QPixmap.
<https://doc.qt.io/qt-6/qicon.html#high-dpi-icons>

Ideally we'd follow the FreeDesktop Icon Theme spec and provide an icon theme for our app that checks all the boxes. <https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html>

Comment (circl-lastname) [reply Ladybird/Qt/ladybird.qrc]: Made an issue for this: #643

**Pull request merged**